apps/emqx/src/emqx_alarm.erl

```diff
@@ -474,3 +476,19 @@ normalize_message(Name, <<"">>) ->
     list_to_binary(io_lib:format("~p", [Name]));
 normalize_message(_Name, Message) ->
     Message.
+
+safe_call(Req) ->
```

Comment:
Given the introduced optimizations in resource_manager (avoid extra activate calls) and emqx_alarm itself (dirty ops instead of transaction), it seems safe to use `gen_server:cast/2` instead. 
However, I'm still reluctant to do it, as cast is easier to abuse.. :thinking: